src/en/getting-started-jaas.md

```diff
@@ -244,9 +244,8 @@ you can manually copy and paste the unique authorisation URL from the command
 output.
 
 After successful authentication, you will be asked to enter a descriptive name
-for the JAAS controller, giving you access to the same controller used by the
-JAAS web interface. This means any models or applications you have already
-deployed are now accessible from the command line.
+for the JAAS controller. This means any models or applications you have
+already deployed are now accessible from the command line.
```

Comment:
Providing a name for the controller does not mean models/applications are now accessible from the CLI. Maybe just:

"Now|Henceforth, any models or applications..."